dont work

daria-naumenko opened the pull request:
Why is it that when I start the application, when I enter the url address into the page, nothing is displayed, even a page not refresh. 
Before that, I had an error **“SEVERE [main] org.apache.catalina.core.StandardServer.await Failed to create server shutdown socket on address [localhost] and port [8005] (base port [8005] and offset [0])”**
In the settings “TomCat” I changed port [8005] on port [8006]

DHrytsuk commented:
> Why is it that when I start the application, when I enter the url address into the page, nothing is displayed, even a page not refresh. Before that, I had an error **“SEVERE [main] org.apache.catalina.core.StandardServer.await Failed to create server shutdown socket on address [localhost] and port [8005] (base port [8005] and offset [0])”** In the settings “TomCat” I changed port [8005] on port [8006]

Does it work now after our discussion and after you changed the port?

daria-naumenko commented:
> > Why is it that when I start the application, when I enter the url address into the page, nothing is displayed, even a page not refresh. Before that, I had an error **“SEVERE [main] org.apache.catalina.core.StandardServer.await Failed to create server shutdown socket on address [localhost] and port [8005] (base port [8005] and offset [0])”** In the settings “TomCat” I changed port [8005] on port [8006]
> 
> Does it work now after our discussion and after you changed the port?

I changed the port to 8006, everything is fine with TomCat, but when I try to write the url on the page, I get a 404 error "JSP file [/WEB-INFstudents.jsp] not found". I changed the file name of student.jsp on students.jsp , but it is unsuccessful

DHrytsuk commented:
> I changed the port to 8006, everything is fine with TomCat, but when I try to write the url on the page, I get a 404 error "JSP file [/WEB-INFstudents.jsp] not found". I changed the file name of student.jsp on students.jsp , but it is unsuccessful

Could you provide the full code (including web.xml and another xml file) to make the full overwie of the configuration

daria-naumenko commented:
> > I changed the port to 8006, everything is fine with TomCat, but when I try to write the url on the page, I get a 404 error "JSP file [/WEB-INFstudents.jsp] not found". I changed the file name of student.jsp on students.jsp , but it is unsuccessful
> 
> Could you provide the full code (including web.xml and another xml file) to make the full overwie of the configuration

In the WEB-INF folder is springServlet-servlet.xml and web.xml, if I understand correctly what you mean

DHrytsuk commented:
> > > I changed the port to 8006, everything is fine with TomCat, but when I try to write the url on the page, I get a 404 error "JSP file [/WEB-INFstudents.jsp] not found". I changed the file name of student.jsp on students.jsp , but it is unsuccessful
> > 
> > 
> > Could you provide the full code (including web.xml and another xml file) to make the full overwie of the configuration
> 
> In the WEB-INF folder is springServlet-servlet.xml and web.xml, if I understand correctly what you mean

Change `<property name="prefix" value="/WEB-INF"/>` to `<property name="prefix" value="/WEB-INF/"/>`
Probably, it could be the reason
